src/App/App.js

```diff
@@ -41,5 +55,4 @@ class App extends Component {
   }
 }
 
-// movieID={props.id}
-export default App;
+export default withRouter(App);
```

Comment:
probably don't need the withRouter on the export of this component, nor the import